File: Bayduk_Yana/lab6/sources/app/src/main/java/com/example/lab6/MainActivity.java
package com.example.lab6;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;


public class MainActivity extends AppCompatActivity {

    private Button next_window_button;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        next_window_button = (Button)findViewById(R.id.button_2);
        this.next_window_button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                String t = "Вы перешли на окно покупки";
                // Уведомление
                Toast.makeText(MainActivity.this, t, Toast.LENGTH_SHORT).show();
                Intent intent = new Intent(MainActivity.this, SecondActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: Gritsak_Alexandra/lab6/sources/app/src/main/java/com/example/lab6/MainActivity.kt
package com.example.lab6

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.content.Intent
import android.widget.Button
import androidx.appcompat.app.AlertDialog
import com.example.lab6.R.string.*



class MainActivity : AppCompatActivity() {

    private lateinit var menuButton:Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        menuButton = findViewById(R.id.button)
        menuButton.setOnClickListener {
            val list: Array<String> = arrayOf(getText(drinks).toString(), getText(salads).toString(),
                getText(desserts).toString())
            var builder: AlertDialog.Builder = AlertDialog.Builder(this)
            builder.setTitle(R.string.message)
            builder.setItems(list) { _, which ->
                val selected = list[which]
                val inet = Intent(this, Dishes::class.java)
                when (selected){
                    "Напитки" -> {
                        inet.putExtra(Dishes.MENU, "Drinks")
                    }
                    "Салаты" -> {
                        inet.putExtra(Dishes.MENU, "Salads")
                    }
                    "Десерты" -> {
                        inet.putExtra(Dishes.MENU, "Desserts")
                    }
                }
                startActivity(inet)
            }
            var dialog = builder.create()
            dialog.show()
        }
    }
}